README.md

```diff
@@ -22,4 +22,39 @@ document](CONTRIBUTING.md).
 Issues can be filled in our [GitHub issue
 tracker](https://github.com/input-output-hk/ouroboros-consensus/issues).
 
+## Installation
 
+
+This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
+
+### Local Development
+
+```
+$ yarn start
```

Comment:
I wonder if we should add a `.nix` file that provides a yarn installation (I could do it in a separate PR). I found it quite handy!

Reply:
Definitely a good idea, and something easy to do (even I could do it :) )!